syntax = "proto3";

import "google/protobuf/empty.proto";

message UserRegisteredNotificationRequest {
  string user_id = 1;

}

message UserActivatedAccountNotificationRequest {
  string user_id = 1;
}


message UserLongNoSeeNotificationRequest {
  string user_id = 1;
}




service UserNotification {

  rpc SendRegistrationNotification (UserRegisteredNotificationRequest) returns (google.protobuf.Empty) {}
  rpc SendActivationNotification (UserActivatedAccountNotificationRequest) returns (google.protobuf.Empty) {}
  rpc SendLongNoSeeNotification (UserLongNoSeeNotificationRequest) returns (google.protobuf.Empty) {}

}